tests/bug-3616-ips/test.yaml

```diff
@@ -0,0 +1,24 @@
+requires:
```

Comment:
I prefer self-speaking names than bug-3616

Can you add README.md files to refer to the redmine issue ?